financial-templates-lib/test/ExpiringMultiPartyClient.js

```diff
@@ -243,6 +218,11 @@ contract("ExpiringMultiPartyClient.js", function(accounts) {
       ],
       client.getUnderCollateralizedPositions(toWei("1.00000000000000001"))
     );
+  });
+
+  it("Returns undisputed liquidations", async function () {
+    await emp.create({ rawValue: toWei("150") }, { rawValue: toWei("100") }, { from: accounts[0] });
+    await emp.create({ rawValue: toWei("1500") }, { rawValue: toWei("100") }, { from: accounts[1] });
```

Comment:
Perhaps a dumb question, but why do we need 2 `create()`'s here? I couldn't get it to work when deleting the second one (with more 1500 synthetic tokens), but wasn't sure why. Anyone have any idea?

Reply:
What goes wrong?

Reply:
@ptare I realize my mistake. If i just comment out line 225 (the second `create()`), then `accounts[1]` not longer has enough synthetic-tokens to call `createLiquidation()` on sponsor 1. So, I replaced the second `create()` with a `syntheticToken.transfer(accounts[1], toWei("100"), { from: accounts[0] } )`.